docs/website/docs/command-reference/list-services.md

```diff
@@ -0,0 +1,52 @@
+---
+title: odo list services
+---
+
+## Description
+
+You can use `odo list services` to list all the bindable Operator backed services on the cluster.
+
+## Running the command
+
+To list bindable services in the current project/namespace:
+```shell
+odo list services
+```
+```shell
+$ odo list services
+ ✓  Listing bindable services from namespace "myproject" [82ms]
+
+ NAME                                               NAMESPACE 
+ redis-standalone/Redis.redis.redis.opstreelabs.in  myproject 
+```
+
+To list bindable services in all projects/namespaces accessible to the user:
+```shell
+odo list services -A 
+```
+```shell
+odo list services -A
+ ✓  Listing bindable services from all namespaces [182ms]
+
+ NAME                                               NAMESPACE  
+ redis-standalone/Redis.redis.redis.opstreelabs.in  myproject  
+ hello-world/RabbitmqCluster.rabbitmq.com           newproject 
+```
+
+To list bindable services in a particular project/namespace that is accessible to the user:
+```shell
+odo list services -n <project-name>
+```
+```shell
+$ odo list services -n newproject
+ ✓  Listing bindable services from namespace "newproject" [45ms]
+
+ NAME                                      NAMESPACE  
+ hello-world/RabbitmqCluster.rabbitmq.com  newproject 
+```
+
+To get the JSON formatted output for any of the above commands, add `-o json` to the commands shown above. That 
```

Comment:
Also please add content in the `json-output.md` file, with an example of output, so the reader can see the structure of the output from the doc.

Reply:
Added.